src/main/ipv-stub/interfaces/ipv-user-identity-response-interface.ts

```diff
@@ -0,0 +1,67 @@
+export interface IpvUserIdentity {
```

Comment:
```suggestion
export interface UserIdentity {
```

This is clear enough I think, no need to specify IPV

Reply:
Could also just call this file user-identity.ts

Reply:
Done

Reply:
Just a comment on this - this is fine but we should follow naming conventions. Where we can group types and interfaces and files are labeled `-types` and `-interfaces` or where we use a more model structure and naming it files with just the object.

Reply:
@isaac-GDS Seeing as this lives in an _interfaces_ dir I personally prefer the latter